plugin/my_asset_manager/MyAssetManagerInterface.py

```diff
@@ -53,6 +59,19 @@ def initialize(self, managerSettings, hostSession):
     def displayName(self):
         return "My Asset Manager"
 
+    def hasCapability(self, capability):
```

Comment:
Yeah I thought about this, but the testing in this repo is more like "Hey look, here's the place where you put your tests", rather than actually having tests. We should rely on API-compliance, (which dosen't yet test HasCapability but it really must soon)